introspection-engine/connectors/mongodb-introspection-connector/tests/test_api/mod.rs

```diff
@@ -121,14 +121,20 @@ where
             database.drop(None).await.unwrap();
         }
 
-        let res = connector.introspect(&datamodel.subject, ctx).await;
+        let res = {
+            let config = psl::parse_configuration(&datamodel_string).unwrap();
+
+            connector
+                .introspect(&datamodel.subject, &config.subject, false, ctx)
+                .await
+        };
```

Comment:
The context arg contains a datasource already, do we need to re-parse the configuration?